packages/las-sdk-core/src/client.ts

```diff
@@ -68,6 +71,43 @@ export class Client {
     this.credentials = credentials;
   }
 
+  /**
+   * Creates an app client, calls the POST /appClients endpoint.
+   *
+   * @param name Name of app client
+   * @param description Description of app client
+   * @returns AppClient response from REST API
+   */
+  async createAppClient(name: string | null, description: string | null): Promise<AppClient> {
+    const body = {
+      name: name,
+      description: description,
+    };
```

Comment:
Guess this could just be
```suggestion
    const body = {
      name,
      description,
    };
```